crates/configuration/src/version3/comparison.rs

```diff
@@ -46,6 +46,11 @@ impl ComparisonOperatorMapping {
                 exposed_name: "_lt".to_string(),
                 operator_kind: OperatorKind::Custom,
             },
+            ComparisonOperatorMapping {
```

Comment:
I think we should avoid making updates to old versions of the configuration format.

In this particular case it probably doesn't matter much either way, since the operator name mapping is just an introspection option, and all we are changing is the default value. Since we stamp out the defaults when we initialize a new configuration only people who explicitly refresh their defaults or create a new configuration with an old version number will ever see these changes.

But generally we should avoid it and prefer users upgrade ther configuration instead. 

In principle this change could have to incur an update of the documentation of the configuration reference section.

Reply:
I agree with you in principle, and changing this won't make a big difference because initialize will always output v5 configs, but it would make me feel better if we do this in this particular case :)